src/libs/actions/ReimbursementAccount/resetFreePlanBankAccount.js

```diff
@@ -44,12 +44,31 @@ function resetFreePlanBankAccount(bankAccountID) {
                     key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
                     value: ReimbursementAccountProps.reimbursementAccountDefaultProps,
                 },
+                {
+                    onyxMethod: CONST.ONYX.METHOD.MERGE,
+                    key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
+                    value: {isLoading: true},
```

Comment:
As a note for myself and reviewers, adding `isLoading` here caused this side effect: https://github.com/Expensify/App/issues/15114

https://github.com/Expensify/App/issues/15114#issuecomment-1430953467